### ISSUE TITLE
Pre-fill Administrator in workshop event

### ISSUE THREAD

**Issue by sheraaronhurt** [carpentries/amy#1767]
Currently, several of the fields from the Centrally-Organised WRF are being automatically pre-filled in the event when accepting the form.  Would like for the "administrator" to be pre-filled with LC, SWC, or DC based on what was selected on the WRF.

**Comment by pbanaszkiewicz** [comment]
@sheraaronhurt please check if this is still valid feature request. I remember I worked on some advancements in prefilling that form, not sure if that was one of them.

**Comment by sheraaronhurt** [comment]
@pbanaszkiewicz  this update didn't happen. This is still a valid feature request.

**Comment by maneesha** [comment]
Yes, in creating a new event from a workshop request/submission, we would want Administrator to auto-populate based on the lesson program of the curricula specified in the workshop request form.

**Comment by pbanaszkiewicz** [comment]
The lessons aren't currently linked to Administrator, so this will require database changes and some migration, and then some changes in form logic.

**Comment by lauryndbrown** [comment]
Just wanted to get some clarity on this one since I'm not as familiar with the website yet.

>Yes, in creating a new event from a workshop request/submission, we would want Administrator to auto-populate based on the lesson program of the curricula specified in the workshop request form.

@maneesha I'm not sure where on the website you can create a new event from a workshop. Could you suggest a url or a screenshot that would help me?

**Comment by pbanaszkiewicz** [comment]
@lauryndbrown I can help you with that.

We store Workshop Requests, Workshop Inquiries and Self-Organised Submissions. All of these come from the forms and are often addressed as WRF - Workshop Request Forms.

You can browse existing entries in `/requests/workshop_requests/` (for Workshop Requests) and `/requests/selforganised_submissions/` (for Self-Organised Submissions). The links to these pages are inside "Requests" dropdown in top-navbar.

Once you open a single request/submission, you'll see a button down below the details table:

![obraz](https://user-images.githubusercontent.com/72821/108958596-79b2a700-7673-11eb-9f70-a614154650a4.png)

It allows you to change state to accepted, or additionally add new event based on the request's data:

![obraz](https://user-images.githubusercontent.com/72821/108958657-918a2b00-7673-11eb-927b-97c51735a44a.png)

This is how the new event from a workshop submission looks like:
![obraz](https://user-images.githubusercontent.com/72821/108958757-b5e60780-7673-11eb-8308-c6bfdd1b57fc.png)

**Comment by maneesha** [comment]
To follow up on Piotr's examples above, here's how we'd want to assign Administrator:

If it's a self-organized submission form: Administrator = self organized.
If it's teaching Data Carpentry, Library Carpentry, or Software Carpentry curriculum, then that should indicate the Administrator.

Thank you - Let me know if you have any other questions!

**Comment by maneesha** [comment]
This is a few years old but my last comment is still accurate with the note that this only applies to events created from a workshop request form.  All other events will have this information filled in manually.

@elichad  I think this should be ready to work on now.

**Comment by elichad** [comment]
related to #2362
